rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -2209,6 +2209,31 @@ void getProposedFederation_whenStorageProviderReturnsProposedFederation_shouldRe
         assertEquals(proposedFederation, actualProposedFederation.get());
     }
 
+    @Test
+    void getProposedFederationCreationBlockNumber_whenStorageProviderReturnsEmpty_shouldReturnErrorCode() {
+        // Act
+        long actualCreationBlockNumber = federationSupport.getProposedFederationCreationBlockNumber();
+
+        // Assert
+        assertEquals(FederationChangeResponseCode.PROPOSED_FEDERATION_NON_EXISTENT.getCode(), actualCreationBlockNumber);
+    }
+
+    @Test
+    void getProposedFederationCreationBlockNumber_whenStorageProviderReturnsProposedFederation_shouldReturnCreationBlockNumber() {
+        // Arrange
+        long creationBlockNumber = 12345L;
+        Federation proposedFederation = P2shErpFederationBuilder.builder()
+            .withCreationBlockNumber(creationBlockNumber)
+            .build();
```

Comment:
same here, the [creation block number](https://github.com/rsksmart/rskj/blob/feat/get_proposed_federation_creation_time_fed_support/rskj-core/src/test/java/co/rsk/peg/federation/P2shErpFederationBuilder.java#L42) is defined by default

Reply:
Sounds good, will udpate

Reply:
To make it more explicit, lets keep it as is